ovos_persona_server/a2a_executor.py

```
@@ -0,0 +1,120 @@
+                [Part(root=TextPart(text="\n".join(response)))],
+                name='full_response',
+            )
+            await updater.complete() # marks task as complete and publishes a final update
+
+        except Exception as e:
+            LOG.error(f'An error occurred while streaming the response: {e}')
+            raise ServerError(error=InternalError()) from e
+
+
+    async def cancel(
+            self, context: RequestContext, event_queue: EventQueue
```

Ruff (0.14.0)

Unused method argument: `context`

(ARG002)